content/en/resources/2024-07-30-demystifying-llms.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Demystifying Llms"
+slug: "demystifying-llms"
+draft: false
+event_date: "2024-07-30T13:00:00-05:00"
+image: "img/resources/webinars/2024-07-30-demystifying-llms.webp"
+name: "Demystifying Hugging Face Models & How to Leverage Them For Business Impact"
+description: "Are you tired of AI hype that promises the world but delivers little? Join Prema Roman and Patrick Deziel for an exclusive, no-nonsense online workshop tailored for data science and product teams who value real results over unrealistic claims. In this workshop, we’ll demonstrate how you can use Hugging Face, one of the most popular open source LLM platforms, to build applications that deliver value to your organization. Video coming soon!"
```

Comment:
The description will display once the webinar ends. It's what users will see until the video is available.